lib/config_hub/client.rb

```diff
@@ -37,6 +37,14 @@ def fetch(key)
       end
     end
 
+    def to_h
+      pull
```

Comment:
@fractallian , question about the use of `pull` here, is this appropriate?

Reply:
no, if config has not been pulled, we should raise a ConfigNotPulledError